app/server/appsmith-server/src/main/java/com/appsmith/server/repositories/ce/CustomActionCollectionRepositoryCEImpl.java

```diff
@@ -177,11 +162,11 @@ public Flux<ActionCollection> findByPageId(String pageId, AclPermission aclPermi
         String unpublishedPage = ActionCollection.Fields.unpublishedCollection_pageId;
         String publishedPage = ActionCollection.Fields.publishedCollection_pageId;
 
-        Criteria pageCriteria = new Criteria()
-                .orOperator(
-                        where(unpublishedPage).is(pageId), where(publishedPage).is(pageId));
+        BridgeQuery<ActionCollection> bridgeQuery = Bridge.or(
+                Bridge.<ActionCollection>equal(unpublishedPage, pageId),
+                Bridge.<ActionCollection>equal(publishedPage, pageId));
```

Comment:
Same as above. The type can be removed. 
``
```suggestion
                Bridge.equal(publishedPage, pageId));
```

Reply:
Done.